cmd/kubeadm/app/constants/constants.go

```diff
@@ -286,7 +286,7 @@ const (
 	MinExternalEtcdVersion = "3.2.18"
 
 	// DefaultEtcdVersion indicates the default etcd version that kubeadm uses
-	DefaultEtcdVersion = "3.4.13-3"
+	DefaultEtcdVersion = "3.5.0-beta.3"
```

Comment:
Line 456-457 need to be updated as well.
```
		22: "3.4.13-3",
		23: "3.4.13-3",
```

Reply:
Done